foundryup/install

```diff
@@ -37,10 +41,28 @@ case $SHELL in
     exit 1
 esac
 
+
 # Only add foundryup if it isn't already in PATH.
 if [[ ":$PATH:" != *":${FOUNDRY_BIN_DIR}:"* ]]; then
-    # Add the foundryup directory to the path and ensure the old PATH variables remain.
-    echo >> $PROFILE && echo "export PATH=\"\$PATH:$FOUNDRY_BIN_DIR\"" >> $PROFILE
+	if [[ $PLATFORM == 'darwin'* ]]; then
+		echo >>"$HOME"/.bash_profile && echo "export PATH=\"\$PATH:$FOUNDRY_BIN_DIR\"" >>"$HOME"/.bash_profile
+	fi
+	# Add the foundryup directory to the path and ensure the old PATH variables remain.
+	echo >>"$PROFILE" && echo "export PATH=\"\$PATH:$FOUNDRY_BIN_DIR\"" >>"$PROFILE"
+fi
+
+
+if [[ $SHELL == 'bash'* && ! -f /usr/local/etc/bash_completion.d ]]; then
+echo "Configuring bash completions for forge and cast..."
+forge completions bash > /usr/local/etc/bash_completion.d/forge
+cast completions bash > /usr/local/etc/bash_completion.d/cast
+
+if [[ $PLATFORM == 'darwin'* ]]; then
+echo "source /usr/local/etc/bash_completion.d/cast" >> ~/.bash_profile
+echo "source /usr/local/etc/bash_completion.d/forge" >> ~/.bash_profile
+fi
+echo "source /usr/local/etc/bash_completion.d/cast" >> ~/.bashrc
+echo "source /usr/local/etc/bash_completion.d/forge" >> ~/.bashrc
```

Comment:
iirc we don't assume a particular shell 

this would force bash, not sure what the convention is for things like that, usually I think programs prompt and ask to do this manually,

wdyt @onbjerg

Reply:
I see what your saying, appreciate the feedback. Ideally this would check the shell env is, however not sure to what extent zsh is being used for osx startup processes